src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleInventoryCleaner.kt

```diff
@@ -166,7 +167,7 @@ object ModuleInventoryCleaner : Module("InventoryCleaner", Category.PLAYER) {
             }
 
             if (executeAction(i, 1, SlotActionType.THROW)) {
-                wait(inventoryConstraints.delay.random())
+                wait { inventoryConstraints.delay.random() }
```

Comment:
This might have changed the way wait works... { ... } instead of ticks as argument.